app/plugins/Wallet/js/lifecycle.js

```diff
@@ -46,8 +48,20 @@ function updateAddrTxn(addr) {
 	}, 'update-history');
 }
 
-// Update wallet summary in header
+// Update transaction history and addresses
 addResultListener('update-status', function(wallet) {
+	refreshRate = finalRefreshRate; // slow down after first successful call
+
+	// Show correct lock status.
+	// TODO: If the wallet is encrypted, prompt with a pw.
```

Comment:
That's an old ToDo, I adjusted the line formatting and moved it to a different location so git shows it as added. 

https://github.com/NebulousLabs/Sia-UI/blob/master/app/plugins/Wallet/js/lifecycle.js#L183